### ISSUE TITLE
Bug: Pressing V while typing on a sign brings up Cheat Menu

### ISSUE THREAD

**Issue by carlosfruitcup** [AlphaVerUnofficialJars/LilypadClient#7]
### Describe the bug

Pressing V while typing on a sign brings up the Cheat Menu leaving the sign UI

As shown in this video
[https://youtu.be/e8QiDQuGLus](url)

### To Reproduce

1. Join Server or Singleplayer World
2. Make or spawn in sign
3. Place the sign anywhere
4. Press V
5. Cheat Menu shows up and sign is placed

**Comment by Smaedd** [comment]
Yeah there's a few instances of things like this, for example you can dash while in chat as well. It's due to the custom alphaver input handler not working the same way as the rest of the input system.
